### PR TITLE
improving build process description

### PR DESCRIPTION
**Pull request by adrianschroeter**

### CODE REVIEW

File: xml/obs_build_process.xml [hunk 1]
```diff
@@ -9,9 +9,10 @@
  xmlns:xlink="http://www.w3.org/1999/xlink" >
  <title>Build Process</title>
  <para>
-  Each package build is created in a fresh environment. This is done to
-  ensure that all dependencies are available and that every later build
-  produces identical results.
+  The build process is the actual process to create out of sources, binaries and
+  config new binaries. This process may run on the &obsa; server side or on a
```
Comment (sknorr) [inline xml/obs_build_process.xml]: Improve word order/reduce sentence length: "The build process creates new binaries from sources, binaries, and config."
(I would expect something that is called "... process" to be "an actual process" in any case. :) )

File: xml/obs_build_process.xml [hunk 2]
```diff
@@ -53,33 +54,34 @@
...
+       Qemu
+      </para>
+     </listitem>
     </itemizedlist> 
     <para>
     In the preinstall phase, the OBS Worker creates a small
-    base system (chroot or VM Image) with manually extracted packages (file
-    system, coreutils, binutils, rpm/debutils, etc.), copies all necessary
-    build requirements into the base system. </para>
+    base system out of the packages declared to be preinstalled (file
```
Comment (sknorr) [inline xml/obs_build_process.xml]: Wording improvement: s/out of/from/

File: xml/obs_build_process.xml [hunk 3]
```diff
@@ -53,33 +54,34 @@
...
+    system, coreutils, binutils, rpm/debutils, etc.). These must be just
+    enough to install further packages. In addition it copies all necessary
```
Comment (sknorr) [inline xml/obs_build_process.xml]: "These must be just enough to install further packages." is a bit unclear -> suggestion: "The tools installed in this phase must only provide the minimum functionality necessary to allow installing further packages."

File: xml/obs_build_process.xml [hunk 4]
```diff
@@ -53,33 +54,34 @@
...
+    build requirements and the source into the base system. </para>
   </sect2>
   <sect2>
    <title>Install Phase</title>
    <para>
-    Depending on the chosen build root, the worker starts a Xen or KVM
-    virtual machine or enters the build root. If this was successful, the
+    Depending on the chosen build environment, the worker may start a
+    virtual machine, an emulator or just enters the build root. If this was successful, the
```
Comment (sknorr) [inline xml/obs_build_process.xml]: Verb congruence: s/enters/enter/

File: xml/obs_build_process.xml [hunk 5]
```diff
@@ -53,33 +54,34 @@
...
    <title>Package Build</title>
-   <remark>
-    FIXME: Terminology: Definition of buildroot? Is buildroot a command? I
-    would have presumed this to be a location not a command or other type of
-    active thingie. - sknorr, 2017-08-24
-   </remark>
    <para>
-    Depending on the type of package, the buildroot executes different build
-    commands:
+    Depending on the type of package, the build environement executes different build
```
Comment (sknorr) [inline xml/obs_build_process.xml]: Typo: s/environement/environment/
But a good fix, thanks!

File: xml/obs_build_process.xml [hunk 6]
```diff
@@ -105,19 +112,17 @@
...
+    additional checks provided via packages. A common toolchain for handling
+    checks is for example <command>rpmlint</command>.
    </para>
   </sect2>
   <sect2>
    <title>After the Build</title>
    <para>
-    The generated packages are taken from the worker, are signed by the
-    &obsa; signer and are published to the Repository.
+     The generated packages get extracted from the build environment and transfered
```
Comment (sknorr) [inline xml/obs_build_process.xml]: * Language: s/get/are/
* Typo: s/transfered/transferred/

File: xml/obs_build_process.xml [hunk 7]
```diff
@@ -105,19 +112,17 @@
...
+     back to the server by the worker. The build result might be post processed by
```
Comment (sknorr) [inline xml/obs_build_process.xml]: Needs to be one word: s/post processed/postprocessed/

**Comment by adrianschroeter** [comment]
@sknorr I thight I followed you in all places

**Pull request merged**